packages/client-api-schema/src/types.ts

```diff
@@ -16,6 +16,18 @@ export type ChannelId = string;
  */
 export type ExternalDestination = string; // currently unused in this schema
 
+/**
+ * Bytes32
+ * @pattern  ^0x([a-fA-F0-9]{64})$
+ */
+export type Bytes32 = string;
+
+/**
+ * Uint256
+ * @pattern  ^0x([a-fA-F0-9]{64})$
```

Comment:
```suggestion
 * @pattern  ^0x([a-fA-F0-9]{0,64})$
```

Reply:
I copied this from the wire-format validator in the xstate wallet.

Reply:
If you have fewer than 64 nibbles, then you have fewer than 256 bits.

And if you have something other than a multiple of 8 nibbles, then you don't have a valid solidity data type at all.

 Of course most things would likely still work regardless, but if we are going to call something a Uint256, it seems to me that it should have 256 bits.